src/SimpleAzure.Storage.HybridQueue/SimpleAzure.Storage.HybridQueue.csproj

```diff
@@ -6,9 +6,9 @@
     <Nullable>enable</Nullable>
     <AssemblyName>WorldDomination.$(MSBuildProjectName)</AssemblyName>
     <RootNamespace>WorldDomination.$(MSBuildProjectName.Replace(" ", "_"))</RootNamespace>
+    <LangVersion>preview</LangVersion>

```

Comment:
will this be part of .NET 8, removing this requirement?

Reply:
If you wanted to stay on C# 12, yes, but as soon as .NET 8 is out, I'd want to start using C# 13 😀

Reply:
Approved until NET8 drops in a few days